src/components/Preview.jsx

```diff
@@ -20,6 +20,7 @@ export default function Preview({
 
   const projectFrames = compiledProjects.map(({source, timestamp}, key) => (
     <PreviewFrame
+      compiledProjectKey={timestamp}
```

Comment:
I agree that the inconsistency of “timestamp” vs. “key” is grating. In fact it does not matter that it’s a timestamp—that is just of a detail of how it is generated. Let’s rename it to `compiledProjectKey` in the record itself.

Reply:
At what point should this happen?

If this is updated in the record should it also be updated in the action, reducer and compiledProjects saga as well

ie.
`const compiledProjectKey = Date.now();` instead of `const timestamp = Date.now();`

Reply:
Yeah I think we should do it as a separate PR, not much advantage in blocking this one.